flambe/dataset/tabular.py

```diff
@@ -217,7 +217,9 @@ def __init__(self,
         self.val_view: Optional[DataView] = None
         self.test_view: Optional[DataView] = None
 
-    def _set_transforms(self, transform: Dict[str, Union[Field, Dict]]) -> None:
+    def _set_transforms(self,
+                        transform: Dict[str, Union[Field, Dict]],
+                        do_setup: bool = True) -> None:
```

Comment:
Add documentation worth it?

Reply:
Not if it remains private. I do think this code needs a bit of refactoring though